docs/how-to/release_process/canary.rst

```diff
@@ -0,0 +1,107 @@
+Canary Testing for Checkbox Edge Version: In-depth Process
+==========================================================
+
+Introduction
+------------
+
+The following sections provide a detailed walkthrough of the Canary Testing process for the Checkbox Edge version, starting from snap building to the outcomes of the testing.
+
+.. toctree::
+   :maxdepth: 1
+
+   canary_pipeline
+   validation_job_example
+   validation_pipeline_execution
+   
+
+Building and Publishing the Checkbox Snap
+-----------------------------------------
+
+Trigger Conditions
+^^^^^^^^^^^^^^^^^^
+The GitHub action responsible for building the snap runs every day. However, it only triggers when the following condition is met:
+
+At least one (PR) has been merged since the last edge build.
```

Comment:
I think "Pull Request" is missing before "(PR)" :)

Reply:
Indeed. Fixed by @tang-mm in the #887.